config/install/tripal.tripalfield_collection.trpcultivate_experiments.yml

```
@@ -1220,3 +1920,235 @@ fields:
             default:
               region: content
               weight: 100
+
+    -   name: bchem_labfacility
+        content_type: biochem_experiment
+        label: Laboratory Facility
+        type: chado_property_type_default
+        description: Include the name or code of the laboratory. Location of the laboratory needs to be included (e.g. City, Province)
```

I'm not sure what is meant by "code" here, and City, Province might be too vague of an example. Lacey and I collaborated on the following suggestion:
```suggestion
        description: Provide the laboratory that this experiment took place in. Include the company name or university, room number and building name, and city and province (e.g. 1A23, Agriculture Building, USASK, Saskatoon, SK)
```